tests/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+project(VectorSimilarity_Benchmark)
+
+cmake_minimum_required(VERSION 3.10)
+cmake_policy(SET CMP0077 NEW)
+set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
+
+set(CMAKE_CXX_STANDARD 14)
+
+if (NOT DEFINED VECSIM_ARCH)
+    set(VECSIM_ARCH "x86-64-v4")
+endif()
+message("VecSim/Benchmark VECSIM_ARCH: " ${VECSIM_ARCH})
+
+if(NOT DEFINED root)
+    get_filename_component(root ${CMAKE_CURRENT_SOURCE_DIR}/../.. ABSOLUTE)
+    set(binroot ${root}/build)
+endif()
+message("VectorSimilarity_Benchmark: " ${binroot})
+
+if(NOT CMAKE_BUILD_TYPE)
+    set(CMAKE_BUILD_TYPE Release)
+endif()
+
+include(FetchContent)
+FetchContent_Declare(
+        googletest
+        # Specify the commit you depend on and update it regularly.
+        URL https://github.com/google/googletest/archive/609281088cfefc76f9d0ce82e1ff6c30cc3591e5.zip
+)
```

Comment:
I know google test is a dependency for google benchmark, but is this fetch required?

Reply:
I guess so, google-benchmark did not built without it..